server/routes/api/form.js

```diff
@@ -0,0 +1,195 @@
+var router = require('express').Router(),
+    fs = require('fs'),
+    path = require('path'),
+    config = require('../../../config/default.js'),
+    authMiddleware = require('../../middleware/auth.js');
+
+function handler(req, res) {
```

Comment:
Yeahhhh, I'm going to but this is really "don't look too closely we are doing some freaky magic"